pkg/datastore/database.go

```diff
@@ -204,27 +200,33 @@ func (db *relationalDb) Find(ctx context.Context, record Record) error {
 	return db.FindInTable(ctx, GetTableName(record), record)
 }
 
+func rollbackTx(tx *gorm.DB, db *relationalDb) {
+	err := tx.Rollback()
```

Comment:
`tx.Rollback` returns `gorm.DB`, not `error`

Reply:
oops missed .Error